packages/contracts-bedrock/test/OptimismMintableERC20Factory.t.sol

```diff
@@ -28,6 +30,24 @@ contract OptimismMintableTokenFactory_Test is Bridge_Initializer {
 
         vm.prank(alice);
         L2TokenFactory.createStandardL2Token(remote, "Beep", "BOOP");
+
+        assertTrue(OptimismMintableERC20(local).decimals() == 18);
+    }
+
+    function test_createStandardL2TokenWithDecimals_succeeds() external {
+        address remote = address(4);
+        address local = calculateTokenAddress(remote, "Beep", "BOOP", 6);
+
+        vm.expectEmit(true, true, true, true);
+        emit StandardL2TokenCreated(remote, local);
+
+        vm.expectEmit(true, true, true, true);
+        emit OptimismMintableERC20Created(local, remote, alice);
+
+        vm.prank(alice);
+        L2TokenFactory.createOptimismMintableERC20WithDecimals(remote, "Beep", "BOOP", 6);
```

Comment:
The create function returns an address, can you add an assertion that the returned address is equal to the calculated address?